EnglishFilter/sections/adservers.txt

```diff
@@ -9,7 +9,6 @@
 ||crevicedepressingpumpkin.com^
 ||peuzubuqo.com^$third-party
 ||viqirue.ru^
-||imuled.work^$third-party
```

Comment:
Unfortunately, not reproduced on my end but there is a good answer https://github.com/AdguardTeam/AdguardFilters/issues/75223#issuecomment-781965506

Reply:
Reproduced it.

Reply:
How am I supposed two write a scriptlet?

Reply:
If you meant popups, @AdamWr fixed it here https://github.com/AdguardTeam/AdguardFilters/commit/af4003af928d82b59a81fc4b3940ffc923a081c5

Reply:
Okay thanks to him.